bin/configure-tower-project.py

```diff
@@ -0,0 +1,455 @@
+                    "workDir": f"s3://{project_stack['TowerBucket']}/work",
+                    "credentials": None,
+                    "computeJobRole": project_stack["TowerForgeBatchWorkJobRoleArn"],
+                    "headJobRole": project_stack["TowerForgeBatchHeadJobRoleArn"],
+                    "headJobCpus": None,
+                    "headJobMemoryMb": None,
+                    "preRunScript": None,
+                    "postRunScript": None,
+                    "cliPath": None,
+                    "forge": {
+                        "vpcId": vpc_stack["VPCId"],
+                        "subnets": [
+                            vpc_stack["PrivateSubnet"],
+                            vpc_stack["PrivateSubnet1"],
+                            vpc_stack["PrivateSubnet2"],
+                            vpc_stack["PrivateSubnet3"],
+                        ],
+                        "fsxMode": "None",
+                        "efsMode": "None",
+                        "type": "SPOT",
+                        "minCpus": 0,
+                        "maxCpus": 100,
+                        "gpuEnabled": False,
+                        "ebsAutoScale": True,
+                        "allowBuckets": [],
+                        "disposeOnDeletion": True,
+                        "instanceTypes": [],
+                        "allocStrategy": None,
+                        "ec2KeyPair": None,
+                        "imageId": None,
+                        "securityGroups": [],
+                        "ebsBlockSize": None,
+                        "fusionEnabled": False,
+                        "efsCreate": False,
+                        "bidPercentage": None,
+                    },
+                },
+            }
+        }
+        response = self.send_tower_request("POST", endpoint, params=params, json=data)
+        return response["computeEnvId"]
+
+    def add_organization_member(self, org_id: str, user: Email) -> int:
+        """Add user to the organization (if need be) and return member ID
+
+        Args:
+            org_id (str): Identifier for the 'Sage Bionetworks' organization
+            user (Email): Email address for the user
+
+        Returns:
+            int: Member ID for the user in the given organization
+        """
+        # Attempt to add the user as a member of the given organization
+        endpoint = f"/orgs/{org_id}/members"
+        data = {"user": user}
+        response = self.send_tower_request(
+            "PUT",
+            f"{endpoint}/add",
+            json=data,
+        )
+        # If the user is already a member, you get the following message:
+        #   "User '<username>' is already a member"
+        # This hacky approach is necessary because you need to retrieve the
+        # member ID using the username (you can't with the email alone)
+        if "message" in response and "already a member" in response["message"]:
+            username = response["message"].split("'")[1]
```

Comment:
The version of Tower that we're running doesn't return the user emails, but I just confirmed that the latest version does on Tower.nf:

```
❯ curl -H "Authorization: Bearer $NXF_TOWER_TOKEN" https://tower.nf/api/orgs/54431251702555/members
{"members":[{"memberId":185620625301515,"userName":"bruno-grande","email":"bruno.grande@sagebase.org","firstName":null,"lastName":null,"avatar":"https://avatars3.githubusercontent.com/u/740725?v=4","role":"owner"}],"totalSize":1}⏎
```

I created a [JIRA ticket](https://sagebionetworks.jira.com/browse/WORKFLOWS-55) for upgrading the version of Tower that we're running because I don't want this PR to grow too much in scope.